gracc-collector.spec

```diff
@@ -7,6 +7,7 @@ License:        MIT
 URL:            https://gracc.opensciencegrid.org
 Source:         %{name}.tar.gz
```

Comment:
I'm not sure how that will work with the auto-build and test though.  What's the best way to do that?